lib/srv/forward/sshserver.go

```diff
@@ -410,10 +409,26 @@ func (s *Server) Serve() {
 		return
 	}
 
+	clusterConfig, err := s.GetAccessPoint().GetClusterConfig()
```

Comment:
you should make this call before dialing to remote client, as you are not cleaning up the connections in this case